examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,200 @@
+# This example demonstrates how to use the NI-Digital Pattern Driver API to create an
+# instrument session, configure various voltage levels, and then burst a
+# pattern. The vol, voh, vih, vil, iol, ioh and vcom values are configurable parameters and the
+# user can select from High-Z (h), Active Load (a) and Three-Level Drive (t)
+# termination options in termSelect.
+#
+# The gRPC API is built from the C API.  NI-Digital Pattern Driver Pattern documentation is installed with
+# the driver at:
+# C:\Users\Public\Documents\National Instruments\NI-Digital-Pattern-Driver\Documentation\Digital Pattern Help.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-Digital Pattern Driver" on the server machine.
+# Link :
+# https://www.ni.com/en-in/support/downloads/drivers/download.ni-digital-pattern-driver.html#367315
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see
+# our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI-Digital Pattern Driver gRPC Wiki for the latest C Function Reference:
+# Link: https://github.com/ni/grpc-device/wiki/NI-DIGITAL-PATTERN-DRIVER-C-Function-Reference
+#
+# Running from command line:
+# Server machine's IP address, port number, and resource name can be passed as separate command line arguments.
+#   > python configure-voltage-levels-and-termination-voltage.py <server_address> <port_number>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763"
+import grpc
+import sys
+import os
+import nidigital_pb2 as nidigital_types
+import nidigital_pb2_grpc as grpc_nidigital
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "NI-Digital-Pattern-Driver-Session"
+
+# resource and options for a simulated 6570 client. Change them according to the NI-Digital Pattern Driver model.
+resource = "PXI1Slot2,PXI1Slot3"
+options = "Simulate=1, DriverSetup=Model:6570"
+# Set dir to point to location of shipped pinmap file with gRPC Example
```

Comment:
@harsha-bhushan dir path location is to be set according to the location of pinmap file shipped with gRPC example as discussed. However, please put your thoughts on this, and let me know what message/comment should we give to user and what should the value for dir path in the example should be.
Accordingly, let me know on the decision of having 2 resources with names given above, or is there anything else that fits better to the decision made.

Reply:
We should ship our own pinmap, spec etc files for this example & not use those from c example. That way we ensure that the user is well aware of what configuration they are using. Hence we could have a single device too in this example as there is no specific reason to include multiple devices. But we need to have good documentation in the file. To summarise:

1. retain PXI1Slot2 & remove the other device in the resources
2. make appropriate change to pinmap file
3. Put a comment above the directory path saying "Provide the absolute path to the folder on the server machine containing the .pinmap, .specs, .digitiming & .digipat files" 
4. Directory path can be an empty string
5. Add a check after this -if directory path is empty & if so show a message on command line - same as the comment in point 3